extensions/widgets/navbar/navbar.lcb

```diff
@@ -256,7 +258,7 @@ navigation bar, the remaining navigation items have their icons reset
 to the default icon.
 
 References: hilitedItemIcons(property), itemLabels(property),
-itemNames(property)
+itemNames(property),  com.livecode.library.iconsvg (library)
 
```

Comment:
This does not parse as an actual link, but does appear in the list of references.